src/core/public/chrome/ui/header/collapsible_nav.test.tsx

```diff
@@ -183,4 +184,28 @@ describe('CollapsibleNav', () => {
     expect(onClose.callCount).toEqual(3);
     expectNavIsClosed(component);
   });
+
+  it('renders custom link above other nav links', () => {
+    const onClose = sinon.spy();
+    const navLinks = [mockLink({ category: kibana }), mockLink({ title: 'categoryless' })];
+    const recentNavLinks = [mockRecentNavLink({})];
+    const customNavLink = mockLink({ title: 'Custom link' });
```

Comment:
nit: I'd probably include this in the above "renders links grouped by category" test as this test doesn't setup anything different and would save some test run time